<ui version="4.0" >
 <class>CustomParameterClass</class>
 <widget class="QWidget" name="CustomParameterClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1087</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>CustomParameter</string>
  </property>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>40</x>
     <y>150</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>参数名：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit" >
   <property name="geometry" >
    <rect>
     <x>100</x>
     <y>150</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>280</x>
     <y>150</y>
     <width>54</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <family>SimSun</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text" >
    <string> =</string>
   </property>
  </widget>
  <widget class="QComboBox" name="paraComboBox1" >
   <property name="geometry" >
    <rect>
     <x>340</x>
     <y>60</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="paraComboBox2" >
   <property name="geometry" >
    <rect>
     <x>540</x>
     <y>60</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="paraComboBox3" >
   <property name="geometry" >
    <rect>
     <x>740</x>
     <y>60</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget1" >
   <property name="geometry" >
    <rect>
     <x>340</x>
     <y>90</y>
     <width>141</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget2" >
   <property name="geometry" >
    <rect>
     <x>540</x>
     <y>90</y>
     <width>141</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget3" >
   <property name="geometry" >
    <rect>
     <x>740</x>
     <y>90</y>
     <width>141</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="operateComboBox1" >
   <property name="geometry" >
    <rect>
     <x>490</x>
     <y>150</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <family>SimSun</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="operateComboBox2" >
   <property name="geometry" >
    <rect>
     <x>690</x>
     <y>150</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <family>SimSun</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="defLineEdit" >
   <property name="geometry" >
    <rect>
     <x>100</x>
     <y>290</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addBtn" >
   <property name="geometry" >
    <rect>
     <x>120</x>
     <y>440</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>添  加</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5" >
   <property name="geometry" >
    <rect>
     <x>41</x>
     <y>250</y>
     <width>77</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>类  型:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="measureLineEdit" >
   <property name="geometry" >
    <rect>
     <x>100</x>
     <y>320</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="typeComboBox" >
   <property name="geometry" >
    <rect>
     <x>99</x>
     <y>250</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6" >
   <property name="geometry" >
    <rect>
     <x>40</x>
     <y>290</y>
     <width>54</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text" >
    <string>定  义:</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar" >
   <property name="geometry" >
    <rect>
     <x>50</x>
     <y>400</y>
     <width>231</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value" >
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_7" >
   <property name="geometry" >
    <rect>
     <x>40</x>
     <y>320</y>
     <width>54</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text" >
    <string>测量方法:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>60</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text" >
    <string>添加参数的相关信息:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="bodyComboBox" >
   <property name="geometry" >
    <rect>
     <x>100</x>
     <y>360</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8" >
   <property name="geometry" >
    <rect>
     <x>40</x>
     <y>360</y>
     <width>54</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text" >
    <string>所属部位:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4" >
   <property name="geometry" >
    <rect>
     <x>41</x>
     <y>210</y>
     <width>77</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>分  组:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="groupComboBox" >
   <property name="geometry" >
    <rect>
     <x>99</x>
     <y>210</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>341</x>
     <y>20</y>
     <width>372</width>
     <height>35</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" >
    <item>
     <widget class="QLabel" name="label_9" >
      <property name="font" >
       <font>
        <family>SimSun</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="text" >
       <string>请选择组合参数的个数：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="paraCountComboBox" >
      <property name="font" >
       <font>
        <family>SimSun</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
